packages/react-components/react-table/src/components/TableCellLayout/renderTableCellLayout.tsx

```diff
@@ -1,16 +1,22 @@
 import * as React from 'react';
 import { getSlots } from '@fluentui/react-utilities';
-import type { TableCellLayoutState, TableCellLayoutSlots } from './TableCellLayout.types';
+import { AvatarContextProvider } from '@fluentui/react-shared-contexts';
+import type { TableCellLayoutState, TableCellLayoutSlots, TableCellLayoutContextValues } from './TableCellLayout.types';
 
 /**
  * Render the final JSX of TableCellLayout
  */
-export const renderTableCellLayout_unstable = (state: TableCellLayoutState) => {
+export const renderTableCellLayout_unstable = (
+  state: TableCellLayoutState,
+  contextValues: TableCellLayoutContextValues,
+) => {
   const { slots, slotProps } = getSlots<TableCellLayoutSlots>(state);
 
   return (
     <slots.root {...slotProps.root}>
-      {slots.media && <slots.media {...slotProps.media} />}
+      <AvatarContextProvider value={contextValues.avatar}>
+        {slots.media && <slots.media {...slotProps.media} />}
+      </AvatarContextProvider>
```

Comment:
Don't need to include the AvatarContextProvider if there's no media content:
```suggestion
      {slots.media && (
        <AvatarContextProvider value={contextValues.avatar}>
          <slots.media {...slotProps.media} />
        </AvatarContextProvider>
      )}
```

Reply:
Fixed in https://github.com/microsoft/fluentui/pull/24807/commits/e29a351b158b30b6f0630eac609c11a9bfedd3bd